crates/pathfinder/src/rpc/serde.rs

```diff
@@ -567,7 +570,10 @@ mod tests {
         );
         assert_eq!(
             bytes_from_hex_str::<32>(TOO_LONG_HEX_STR),
-            Err(HexParseError::InvalidLength(65))
+            Err(HexParseError::InvalidLength {
+                max: 64,
+                actual: 65
+            })
         );
     }
```

Comment:
```suggestion
        // Regression: previously max in the error message was hard-coded at 64,
        // so try another buf size to make sure it is not anymore
        assert_eq!(
            &format!("{}", bytes_from_hex_str::<1>("abc").unwrap_err()),
            "More than 2 digits found: 3"
        );
    }
```

Reply:
Oops already merged lol...

Reply:
Yeah my bad.

Reply:
No problemo, it's already shipped, I can add this in a new micro pr ;) .